packages/phone-number-privacy/monitor/src/query.ts

```diff
@@ -36,12 +49,33 @@ export const queryOdisForSalt = async () => {
     phoneNumber,
     accountAddress,
     authSigner,
-    OdisUtils.Query.getServiceContext(network() as OdisContextName),
+    serviceContext,
     undefined,
     undefined,
     undefined,
     genSessionID(),
     undefined,
-    undefined
+    endpoint
   )
 }
+
+export const queryOdisDomain = async (contextName: OdisContextName) => {
+  console.log(`contextName: ${contextName}`) // tslint:disable-line:no-console
+
+  const serviceContext = getServiceContext(contextName, OdisAPI.DOMAIN)
+  const monitorDomainConfig: OdisHardeningConfig = {
+    rateLimit: [
+      {
+        delay: 0,
+        resetTimer: defined(true),
+        // Running every 5 min, this should not run out for the next 9 million years
+        batchSize: defined(1000000000000),
+        repetitions: defined(1000000000000),
+      },
+    ],
+    environment: serviceContext,
+  }
+  const authorizer = odisQueryAuthorizer(Buffer.from('ODIS domains monitor authorizer test seed'))
+  const domain = buildOdisDomain(monitorDomainConfig, authorizer.address)
+  return odisHardenKey(Buffer.from('password'), domain, serviceContext, authorizer.wallet)
```

Comment:
might be worth having a comment explaining that this throws if the signature verification fails